README.md

```diff
@@ -7,23 +7,35 @@
 [![Dependency Status](https://gemnasium.com/sensu-plugins/sensu-plugins-windows.svg)](https://gemnasium.com/sensu-plugins/sensu-plugins-windows)
 
 ## Functionality
+These files provide basic Checks and Metrics for a Windows system.
 
 ## Files
- * bin/metrics-windows-cpu-load.rb
- * bin/metrics-windows-disk-usage.rb
- * bin/metrics-windows-network.rb
- * bin/metrics-windows-ram-usage.rb
- * bin/metrics-windows-uptime.rb
- * bin/check-windows-cpu-load.rb
- * bin/check-windows-disk.rb
- * bin/check-windows-process.rb
- * bin/check-windows-ram.rb
- * bin/check-windows-service.rb
+ * bin/ruby/check-windows-cpu-load.rb
+ * bin/ruby/check-windows-disk.rb
+ * bin/ruby/check-windows-process.rb
+ * bin/ruby/check-windows-ram.rb
+ * bin/ruby/check-windows-service.rb
+ * bin/ruby/metric-windows-cpu-load.rb
```

Comment:
Why not call them checks instead of check?  Having verbal conversations with other techs can make things very confusing if you're switching plurality constantly.  It just makes it easier to convey the meaning during discussions.